### DISCUSSION TITLE
Add password's entropy score

### DISCUSSION THREAD

**Discussion by Ozwel** [keepassxreboot/keepassxc#8933] [Ideas]
Hi,

KeePass shows password's entropy in every entry. It's quite convenient to check quickly how strong a password is.

KeePassXC only shows entropy while creating a password in the password generator. Is it possible to show entropy in records and not only in the generator?

**Comment by michaelk83** [comment]
Already done. Upgrade to the latest version.
#7885

**Comment by Ozwel** [comment]
I can't see the number of bits, only a colored line. I was talking about the number of bits of entropy which we can see when generating a password. I'm using the last version of KeePassXC on macOS which is 2.7.4.

**Reply by michaelk83** [reply]
The precise entropy number is not important, since you get diminishing returns after some minimum. As long as the password is considered "Strong", that's good enough. You can tell that at a glance by the length and color of that line, or with the tooltip when you hover over it.

**Reply by droidmonkey** [reply]
The number of bits shows to the right of the line. Not sure why you are not seeing it.

**Comment by Ozwel** [comment]
I can see it in the generator, not in the "edit entry" window.
